src/items/racial-features/thermal_consumption.json

```diff
@@ -0,0 +1,167 @@
+{
+  "_id": "PrFB5DUyacPMGirw",
+  "name": "Thermal Consumption",
+  "type": "feat",
+  "img": "icons/svg/mystery-man.svg",
+  "system": {
+    "type": "",
+    "ability": null,
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "",
+      "condition": "",
+      "cost": 0
+    },
+    "area": {
+      "effect": "",
+      "shapable": false,
+      "shape": "",
+      "units": "",
+      "value": 0
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>Thyrs consume thermal energy. They have fire resistance 5, which stacks with one other source of fire resistance. Once per day as a reaction when a thyr takes fire damage, they can increase their fire resistance by 5 until the beginning of their next turn. Until the end of their next turn, all of the thyr’s speeds increase by 30 feet, to a maximum of twice their normal speed using that form of movement. This increase counts as an enhancement bonus.</p>\n<p><em>System does not currently support stacked Energy Resistance bonuses. Emulate stacking by editing the modifier in the Modifier tab.</em></p>\n<p><em>Enable/Disable Reaction bonuses in the Modifiers tab.</em></p>"
```

Comment:
Can't this just be an active feat?

Reply:
No, because the 1st Fire resistance 5 is a passive effect. It's the raising it to 10, and the speed bonus that is active.